deploy/cypress_setup.py

```diff
@@ -20,7 +21,32 @@ def cypress(profile: str, schema: str):
     """
     )
 
-    conn.close()
+    start_time = time.time()
+
+    while True:
+        try:
+            # Execute a query to fetch data from the table
+            cur.execute(
+                "SELECT * FROM internal.config where key in ('WAREHOUSE_EVENTS_MAINTENANCE', 'QUERY_HISTORY_MAINTENANCE');"
```

Comment:
Can't comment on correctness of the check (don't know if say value could be null and it means we are not setup correctly yet). Maybe we should protect against that? Or empty string?

`SELECT count(*) FROM internal.config where key in ('WAREHOUSE_EVENTS_MAINTENANCE', 'QUERY_HISTORY_MAINTENANCE') and value is not null ;`